app/views/publishers/vacancies/job_applications.html.slim

```diff
@@ -0,0 +1,74 @@
+- content_for :page_title_prefix, current_organisation.name
+
+= render BannerComponent.new do
+  - if deadline_passed?(@vacancy.expires_at)
+    = govuk_breadcrumbs breadcrumbs: { "#{t("publishers.vacancies_component.expired.tab_heading")}": jobs_with_type_organisation_url(:expired), "#{@vacancy.job_title}": organisation_job_job_applications_url(@vacancy.id) }
+  - else
+    = govuk_breadcrumbs breadcrumbs: { "#{t("publishers.vacancies_component.published.tab_heading")}": organisation_url, "#{@vacancy.job_title}": organisation_job_job_applications_url(@vacancy.id) }
+  .govuk-grid-row
+    .govuk-grid-column-full
+      h2.govuk-caption-l = t(".heading")
+      h1.govuk-heading-l = @vacancy.job_title
+
+  .govuk-grid-row
+    .govuk-grid-column-one-quarter
+      .application-deadline class=("application-deadline--passed" if deadline_passed?(@vacancy.expires_at))
+        h4.govuk-body.govuk-caption-s class="govuk-!-font-weight-bold govuk-!-margin-bottom-0"
+          = deadline_passed?(@vacancy.expires_at) ? t(".deadline.after") : t(".deadline.before")
+        span.govuk-body = expiry_date_and_time(@vacancy)
+
+    .govuk-grid-column-one-quarter
+      .application-count.govuk-tag class="govuk-tag--#{status_tag_colour(:submitted)}"
```

Comment:
Do we need a helper method for this? 

If we do, should we pass in a symbol and convert it to a string or should we pass in a string in the first place?